jekyll/_cci2/language-java-maven.md

```diff
@@ -59,7 +59,7 @@ version: 2.0
 jobs:
   build:
     docker:
-      - image: circleci/openjdk:stretch
+      - image: cimg/openjdk:stretch
```

Comment:
@thomasr8 there is no stretch variant for openjdk: https://circleci.com/developer/images/image/cimg/openjdk

@felicianotech any advice on what we should be replacing stretch variants with in examples?

Reply:
Added ":17.0" based on @felicianotech feedback.